recipe/meta.yaml

```diff
@@ -1,16 +1,13 @@
 {% set name = "jedi" %}
-{% set version = "0.9.0" %}
-{% set hash_type = "sha256" %}
-{% set hash = "3b4c19fba31bdead9ab7350fb9fa7c914c59b0a807dcdd5c00a05feb85491d31" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  git_url: https://github.com/davidhalter/jedi.git
+  git_tag: v{{ version }}
```

Comment:
github generates tarballs, here is the URL: https://github.com/davidhalter/jedi/archive/v0.10.0.tar.gz